devicon.json

```
@@ -8381,6 +8381,29 @@
             }
         ]
     },
+    {
+        "name": "reactrouter", 
+        "altnames": [
+            "reactrouter"
+        ],
```

If the alternative name's the same as the name, it shouldn't be added.
```suggestion
        "altnames": [],
```